readme.md

```diff
@@ -177,6 +179,31 @@ the anecdote, you can throw away the data."
 [R: verywell.com](https://www.verywell.com/what-is-the-hawthorne-effect-2795234)
 
 
+#### [Munchausen syndrome](https://en.wikipedia.org/wiki/Factitious_disorder_imposed_on_self)
+
+> Person repeatedly and deliberately acts as if he or she has a physical or mental illness when he or she is not really sick.
+
+![Image](./img/Munchausen_syndrome.png)
+
+Munchausen's syndrome is a psychological and behavioural condition where someone pretends to be ill or induces symptoms of illness in themselves. It is also sometimes known as factitious disorder. People with the condition intentionally produce or pretend to have physical or psychological symptoms of illness. Their main intention is to assume the "sick role" to have people care for them and be the centre of attention. 
```

Comment:
Wrap at 70

Reply:
Sorry, I didn't get you.
Wrap at 70?

Should I shorten the paragraph?

Reply:
I shortened it.